# Get a trajectory of arm poses based on the trajectory name
string name
---
ArmPose[] trajectory
